src/main/java/com/epam/aidial/core/upstream/UpstreamRoute.java

```diff
@@ -126,6 +126,7 @@ private static long calculateRetryAfterSeconds(HttpClientResponse response) {
             if (retryAfterHeaderValue != null) {
                 return Long.parseLong(retryAfterHeaderValue);
             }
+            log.info("Retry-after header not found, available headers: {}", response.headers());
```

Comment:
Please don’t print all headers. There might be sensitive info like api key or jet

Reply:
fixed